README.md

```diff
@@ -555,31 +555,17 @@ pages that are hosted by GitHub and can easily be published via
 GitHub.  You can post the website for your Maven projects by first
 creating a `gh-pages` branch (from the initial revision of the 
 repository), removing the `README.md` file on that branch (because
-it will take precedence over the generated `index.html` file) and
-pushing it to GitHub:
+it will take precedence over the generated `index.html` file), 
+removing `*.zip` from `.gitignore` (because the Maven process will add
```

Comment:
Could you move this change and the addition of the script to another Pull Request?  I want to think about them some more.